WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -2,634 +2,677 @@
+                        android:layout_width="match_parent"
                         android:layout_height="@dimen/list_divider_height"
+                        android:layout_below="@+id/quick_start_customize"
                         android:layout_marginStart="@dimen/quick_start_type_divider_margin_start"
-                        android:layout_width="match_parent" >
+                        android:background="@color/neutral_0">
                     </View>
 
                     <RelativeLayout
                         android:id="@+id/quick_start_grow"
-                        android:layout_below="@+id/quick_start_customize"
-                        style="@style/QuickStartTypeRow" >
+                        style="@style/QuickStartTypeRow"
+                        android:layout_below="@+id/quick_start_customize">
 
                         <ImageView
                             android:id="@+id/quick_start_grow_icon"
+                            style="@style/QuickStartTypeIcon"
                             android:background="@drawable/bg_oval_pink_50_multiple_users_white_40dp_selector"
-                            android:importantForAccessibility="no"
-                            style="@style/QuickStartTypeIcon" >
+                            android:importantForAccessibility="no">
                         </ImageView>
 
                         <TextView
                             android:id="@+id/quick_start_grow_title"
+                            style="@style/QuickStartTypeTitle"
                             android:layout_toEndOf="@+id/quick_start_grow_icon"
-                            android:text="@string/quick_start_sites_type_grow"
-                            style="@style/QuickStartTypeTitle" >
+                            android:text="@string/quick_start_sites_type_grow">
                         </TextView>
 
                         <TextView
                             android:id="@+id/quick_start_grow_subtitle"
+                            style="@style/QuickStartTypeSubtitle"
                             android:layout_below="@+id/quick_start_grow_title"
                             android:layout_toEndOf="@+id/quick_start_grow_icon"
-                            tools:text="0 of 5 complete"
-                            style="@style/QuickStartTypeSubtitle" >
+                            tools:text="0 of 5 complete">
                         </TextView>
 
                     </RelativeLayout>
 
                 </RelativeLayout>
 
-            </androidx.cardview.widget.CardView>
+                <!--Register Domain-->
+                <LinearLayout
+                    android:id="@+id/my_site_register_domain_cta"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="@dimen/margin_large"
+                    android:background="@color/white"
```

Comment:
```suggestion
                    android:background="@android:color/white"
```

Can we change this to `@android:color/white`?  There is no resource `white` in `colors.xml` so using `@color/white` would be pulling from another source or library.